src/app/app.component.ts

```diff
@@ -41,6 +45,10 @@ export class MyApp {
   openPage(page) {
     // Reset the content nav to have just this page
     // we wouldn't want the back button to show in this scenario
+    if(page.title == 'Deconnexion'){
+      this.userServices.token = '';
+    }
+    console.log('le token est :'+this.userServices.token);
     this.nav.setRoot(page.component);
```

Comment:
Do you really want to set root for any pages? If i'm in list and going on my profile, i would like to be able to go back on the list for example.

Reply:
this is needed as the navigation is in the appmodule and we cannot declare a navcontroller there. 
The only difference for the user will concern its profile as he will have to pass through the menu to go back. Beside that, this allows us to launch the login screen safely.
If we really want to control all the aspects of the navigation, it will be a hell lot more trouble. Small change, big difference

Reply:
ok, with explanation it's clearer for me